app/helpers/application_helper.rb

```diff
@@ -121,6 +121,17 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon, attributes = {})
+    tag.i(
+      asset_content("#{icon}.svg"),
+      class: %w(icon).concat(attributes[:class].to_s.split)
+    )
+  end
+
+  def asset_content(path)
+    Rails.application.assets.load_path.find(path).content.html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
This means we read the file from disk on each icon render? 

`inline_svg` has a way to load them all in memory on server start, but it might not work well for us as most icons are only used in the frontend.

Reply:
I know propshaft caches the results of it's path loading/searching process, not sure if it caches the file contents themselves, or if that's a file read every time.

At minimum, we could do a lightweight hash memoization (using icon names as keys) in this method, so we'd have one read per icon per request instead of repeated ... not sure about the boot time approach.

Will do a litle more propshaft research to see what is/isnt cached already there.